README.md

```diff
@@ -61,7 +61,9 @@ linking Ewon devices using a direct data path with a Flexy Java application.
         - [Configured Logging Level](#configured-logging-level)
         - [Logging Performance](#logging-performance)
         - [Adding Log Output](#adding-log-output)
-- [Inventory Object](#inventory-object)
```

Comment:
This commit looks like it should be merged with the previous one

Reply:
yeah, looks like I decided to make object plural

Reply:
Its not just the pluralization. [Inventory Object](#inventory-object) gets added in the previous commit then deleted here. Then TOC items are added in this commit for entries that were added in the last commit.